src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -1,4 +1,16 @@
 <?php
+/**
+ * This file handles the UserPageDecorationMiddleware for LORIS
+ *
+ * PHP Version 7
+ *
+ * @category LORIS
+ * @package  Main
+ * @author   Dave MacFarlane <driusan@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
```

Comment:
This doesn't have anything to do with PHP8 and looks like a step backwards in terms of our phpcs standards..